cli/scorecard/violations_test.go

```diff
@@ -126,3 +126,47 @@ func TestGetViolations(t *testing.T) {
 		})
 	}
 }
+
+func TestGetViolationsConcurrently(t *testing.T) {
```

Comment:
Same comment as earlier: we should avoid simply copying the code from above and instead share most of the test logic.